examples/mnist/README.md

```diff
@@ -34,32 +34,55 @@ class ModelOwner:
 class PredictionClient:
 
     def provide_input(self) -> tf.Tensor:
-        prediction_input = self._build_data_pipeline().get_next()
-        # pre-processing
-        prediction_input = tf.reshape(prediction_input, shape=(self.BATCH_SIZE, 28 * 28))
+        """Prepare input data for prediction."""
+        with tf.name_scope('loading'):
+        prediction_input, expected_result = self._build_data_pipeline().get_next()
+        print_op = tf.print("Expect", expected_result, summarize=self.BATCH_SIZE)
+        with tf.control_dependencies([print_op]):
```

Comment:
for this section, I would remove `tf.name_scope`, print_op, `tf.control_dependencies`. It's too many details for the  README. I would suggest to match the same level of details as before.